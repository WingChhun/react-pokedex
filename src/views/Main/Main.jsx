import React, { Component } from 'react';
import PropTypes from 'prop-types';
import styled from 'styled-components';
import memoize from 'memoize-one';




/**
 * todo: Hook up Context that queries for json
 * todo: Build layout using styled components
 * todo: Minimal responsive touchup
 * todo: performance optimization
 */
class Main extends Component {

  render() {
    return (
      <div>
        Main view
      </div>
    );
  }
}


export default Main;