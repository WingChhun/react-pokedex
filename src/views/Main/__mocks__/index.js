

//PokemonContext
const PokemonContext = {


}

export PokemonContext;