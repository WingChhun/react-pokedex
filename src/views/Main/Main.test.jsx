import React from 'react';
import Main from "./Main";
import { shallow } from 'enzyme';


describe(`<Main/>`, () => {

  it(`renders properly`, () => {
    const wrapper = shallow(<Main />);
    expect(wrapper).toMatchSnapshot();
  });
});