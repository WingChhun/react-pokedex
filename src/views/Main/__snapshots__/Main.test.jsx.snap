// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main/> renders properly 1`] = `ShallowWrapper {}`;
